lib/orwell/web/templates/layout/app.html.eex

```diff
@@ -15,12 +15,15 @@
     <div class="container">
       <header class="header">
         <h1>Orwell</h1>
+        <nav>
```

Comment:
We should consider moving this to slim (I know this might not be the correct PR for it)